### ISSUE TITLE
Visual Studio ContentDialog template creates dialog with old uwp styles (non WinUI 2.6)

### ISSUE THREAD

**Issue by movpushmov** [microsoft/microsoft-ui-xaml#5500]
**Describe the bug**
If you create custom ContentDialog with VS template then you can see old UWP styles instead of new WinUI 2.6 styles.

**Steps to reproduce the bug**
1. Open Project
2. Create ContentDialog with VS template in separated file
3. Call ShowAsync:

```csharp
new SomeContentDialogClass().ShowAsync();
```

4. See old ContentDialog styles.

**Expected behavior**
ContentDialog must show new WinUI 2.6 styles

**Screenshots**

***Custom ContentDialog***
![image](https://user-images.githubusercontent.com/49155506/126067147-e4046cd1-661b-49a0-a8af-31d3690f44c0.png)

***ContentDialog created with this code:***
```csharp
new ContentDialog().ShowAsync();
```
![image](https://user-images.githubusercontent.com/49155506/126067295-ee5bf60c-352f-4dc2-a63d-fa737af38448.png)

**Version Info**
WinUI 2.6

NuGet package version: 
[Microsoft.UI.Xaml 2.6.0]

Windows app type:
| UWP              | Win32            |
| :--------------- | :--------------- |
| Yes |  |

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              |    |
| October 2020 Update (19042)        | Yes   |
| May 2020 Update (19041)            |    |
| November 2019 Update (18363)       |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :----------------- | :--------------- |
| Desktop            | Yes |
| Xbox               |  |
| Surface Hub        |  |
| IoT                |  |

**Comment by StephenLPeters** [comment]
Can you ellaborate on "Create ContentDialog with VS template in separated file" means? I'm not aware of it and a quick google search doesn't seem to say.

**Comment by crramirez** [comment]
Hello @StephenLPeters ,

More context:
https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.contentdialog?view=winrt-20348#derived-controls-with-winui-styles
![image](https://user-images.githubusercontent.com/8397379/126454151-b634891c-db96-4ea0-b321-57df86014b29.png)

VisualStudio templates for create ContentDialog, creates a derived control **without** WinUI styles

Yesterday I had to add this to all of my dialogs that were created with VS templates

Regards

**Comment by movpushmov** [comment]
> Hello @StephenLPeters ,
> 
> More context:
> https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.contentdialog?view=winrt-20348#derived-controls-with-winui-styles
> ![image](https://user-images.githubusercontent.com/8397379/126454151-b634891c-db96-4ea0-b321-57df86014b29.png)
> 
> VisualStudio templates for create ContentDialog, creates a derived control **without** WinUI styles
> 
> Yesterday I had to add this to all of my dialogs that were created with VS templates
> 
> Regards

This code works, thanks!

**Comment by crramirez** [comment]
I only provided the workaround, the issue still exists, and any user who uses the templates will face the same. In my opinion this issue shouldn't be closed yet.

**Comment by StephenLPeters** [comment]
This behavior is how the system is designed.  Winui2 is just a NuGet package built on top of system xaml that provides new controls and new styles for old controls.  Like any other style package, we cannot change the default style of the build in controls,  thus if you want to provide our styles to your own controls, you'll need to do it explicitly since they are not the default style.

**Comment by crramirez** [comment]
@StephenLPeters I agree with you. If there are templates to create WinUI 2 and WinUI 3 applications with the included NuGet package, why do not have a template to create a WinUI 2 ContentDialog that includes the code above? so the developer doesn't need to apply the styles over and over.

I understand that the standard and the recommended way is now WinUI 2 or WinUI 3, there is no point to continue having templates that by default do not rely on the new styles. Even if the system is designed in this way, this doesn't mean that this complexity must be managed entirely by the developer.

**Comment by StephenLPeters** [comment]
Sure,  @anawishnoff FYI

**Comment by movpushmov** [comment]
@StephenLPeters any changes?

**Comment by nickrandolph** [comment]
Ping - any update on this. Seems unnecessary for developers to have to worry about this - WinAppSdk apps should look awesome out the box!

**Comment by DmitriyKomin** [comment]
Duplicate of #8869
